guides/concepts/list-of-operations.md

```diff
@@ -77,19 +77,65 @@ Possible errors:
 |PAYMENT_LINE_FULL| -8| The destination account (receiver) does not have sufficient limits to receive `amount` and still satisfy its buying liabilities.|
 |PAYMENT_NO_ISSUER| -9| The issuer of the asset does not exist.|
 
-## Path Payment
-[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.pathPayment) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/PathPaymentOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#PathPayment)
+## Path Payment Strict Send
+[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.pathPaymentStrictSend) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/PathPaymentStrictSendOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#PathPaymentStrictSend)
 
-Sends an amount in a specific asset to a destination account through a path of offers. This allows the asset sent (e.g., 450 XLM) to be different from the asset received (e.g, 6 BTC).
+A path payment sends an amount of a specific asset to a destination account through a path of offers. Since the asset sent (e.g., 450 XLM) can be different from the asset received (e.g, 6 BTC), path payments allow for the simultaneous transfer and conversion of currencies.  
+
+A Path Payment Strict Send allows a user to specify the *amount of the asset to send*. The amount received will vary based on offers in the order books.  If you would like to instead specify the amount received, use the [Path Payment Strict Receive](#path-payment-strict-receive) operation.  
+
+A few things to note:
+* path payments don't allow intermediate offers to be from the source account as this would yield a worse exchange rate. You'll need to either split the path payment into two smaller path payments, or ensure that the source account's offers are not at the top of the order book.
+* balances are settled at the very end of the operation
+   * this is especially important when `(Destination, Destination Asset) == (Source, Send Asset)` as this provides a functionality equivalent to getting a no interest loan for the duration of the operation.
+
+Threshold: Medium
+
+Result: `PathPaymentStrictSendResult`
+
+Parameters:
+
+|Parameters| Type| Description|
```

Comment:
Consider for a future PR: it's nice to add spacing to the tables so that they are readable in raw markdown. You can use e.g. http://markdowntable.com/ to produce it quickly.